.github/workflows/main.yml

```diff
@@ -206,10 +206,13 @@ jobs:
         focus: [""]
         bucket: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
         include:
-          - k8s_version: v1.21.10
+          - k8s_version: v1.22.10
```

Comment:
It looks like v1.22.9 is the newest v1.22 with a kind image published: https://hub.docker.com/r/kindest/node/tags?page=1&name=v1.22